adapters/e_volution/e_volution.go

```diff
@@ -0,0 +1,120 @@
+package e_volution
```

Comment:
It's best practice in Go to only use alphanumeric characters in package names. Please rename the folder to evolution, the package to evolution, and the main files to evolution.go and evolution_test.go for consistency.